a3p-integration/proposals/p:upgrade-19/replaceFeeDistributor.test.js

```diff
@@ -33,6 +33,9 @@ import {
 import { AmountMath } from '@agoric/ertp';
 import { floorMultiplyBy } from '@agoric/zoe/src/contractSupport/ratio.js';
 
+// TODO @import {Ratio} from '@agoric/zoe'
+/** @typedef {any} Ratio */
```

Comment:
It's in ERTP now. Why wait?
```suggestion
/** @import {Ratio} from '@agoric/ertp`;
```
(it might need src/types.js)

Reply:
```
error TS2305: Module '"@agoric/ertp"' has no exported member 'Ratio'.
```
```
error TS2305: Module '"@agoric/ertp/src/types.js"' has no exported member 'Ratio'.
```

Reply:
Ah, the proposal must have an old version of ERTP. We'd have the one in agoric-sdk if this used the new 'yarn link' trick but it's not worth further churn on the PR.